### PR TITLE
Changed redirect and updated title for ref importer

### PR DESCRIPTION
**Pull request by gammonpeter**

### CODE REVIEW

File: src/app/import-models/import-models.component.ts [hunk 1]
```diff
@@ -69,40 +70,54 @@ export class ImportModelsComponent implements OnInit {
     private stateHandler: StateHandlerService,
     private broadcastSvc: BroadcastService,
     private stateService: StateService
-  ) { }
+  ) {}
 
   ngOnInit() {
     // tslint:disable-next-line: deprecation
-    this.importType = this.stateService.params.importType ? this.stateService.params.importType : 'dataModels';
+    this.importType = this.stateService.params.importType
```
Comment (pjmonks) [inline src/app/import-models/import-models.component.ts:77]: While modifying this area, could you also replace the deprecated `stateService.params` with `uiRouterGlobal.params` please?

**Review by pjmonks — CHANGES_REQUESTED** [review]
The actual fix looks good. I left one comment to tidy up the changes a bit more, if you could do that please then I'll merge for you.

File: src/app/import-models/import-models.component.ts (hunk 1 above)
Comment (gammonpeter) [reply src/app/import-models/import-models.component.ts]: Updated to use UiRouterGlobals

**Review by pjmonks — APPROVED** [review]
All good

**Pull request merged**